.github/workflows/spread.yaml

```diff
@@ -31,7 +31,9 @@ jobs:
       fail-fast: false
       matrix:
         spread:
```

Comment:
Would a reference in the other direction also be useful?
```suggestion
        # from the backends defined in the top-level `spread.yaml`
        spread:
```

Reply:
Good idea!